bgc_data/bgc_picoplankton.dataresource.yaml

```diff
@@ -10,6 +10,10 @@ layout:
   skipFields: [ FID ]
 schema:
   fields:
+  - name: PROJECT
```

Comment:
There is already a `PROJECT` field in bgc_trip, and each `TRIP_CODE` is associated with a single `PROJECT`. This change breaks that model. If there are currently multiple rows of data with the same `TRIP_CODE` and `SAMPLEDEPTH_M`, maybe they (or just the SARDI data) should be assigned a slightly different `TRIP_CODE` and then that can be done in the bgc_trip table.

On a practical note, as it currently stands, that currently has project "SARDI" in this table will not be included in any of our output products, because
* The combined product now explicitly includes "NRS" data only (see SQL change below); and
* The raw picoplankton only sub-products (`bgc_picoplankton_data`) is based on the bgc_picoplankton_meta table, not this one.

Reply:
It looks to me like this is only an issue are for "KAI" trips, where "SARDI" rows only have virus data, while the "NRS" rows only have other data, not virus. So potentially those could be rolled into one (using a group by TRIP_CODE and SAMPLEDEPTH_M) without any loss of data? Would that work?

Reply:
I am happy to update GeoServer and remove the TRIP_CODE, however the remaining (candidate) primary key fields TRIP_CODE and SAMPLEDEPTH_M are not unique. They are not even close to being unique!
eg.
```
select *
from bgc_picoplankton
where project = 'Coastal Stations' and sampledepth_m = '13';
```

```PROJECT           TRIP_CODE                       SAMPLEDEPTH_M  SAMPLEDATELOCAL      PROCHLOROCOCCUS_CELLSML  SYNECOCHOCCUS_CELLSML  PICOEUKARYOTES_CELLSML  BACTERIA_CELLSML  VIRUS_CELLSML  PROCHLOROCOCCUS_FLAG  SYNECOCHOCCUS_FLAG  PICOEUKARYOTES_FLAG  BACTERIA_FLAG  VIRUS_FLAG  
----------------  ------------------------------  -------------  -------------------  -----------------------  ---------------------  ----------------------  ----------------  -------------  --------------------  ------------------  -------------------  -------------  ----------  
Coastal Stations  PTD20200929_102.100.100/138884  13             2020-09-29 13:05:00  68189                    49504                  4166                    611150            (null)         1                     1                   1                    1              (null)      
Coastal Stations  PTD20200929_102.100.100/138885  13             2020-09-29 13:05:00  68419                    47691                  3370                    539963            (null)         1                     1                   1                    1              (null)      
Coastal Stations  PTD20200929_102.100.100/138886  13             2020-09-29 13:05:00  72521                    54949                  4393                    643930            (null)         1                     1                   1                    1              (null)      ```

Reply:
Oops - @clairedavies just reminded me that primary keys should be trip_code and sampledepth_m, which works a lot better! : )  I'm just putting those changes in now, I'll add to this PR if that works for everyone.